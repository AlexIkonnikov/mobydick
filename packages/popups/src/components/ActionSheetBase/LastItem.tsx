import React, {FC} from 'react';
import {useStyles} from '@npm/mobydick-styles';

import stylesCreate from './stylesCreate';
import Item, {IPropsItem} from './Item';

const LastItem: FC<IPropsItem> = props => {
  const [styles] = useStyles(stylesCreate);

  return <Item style={styles.lastItem} {...props} />;
};

export default LastItem;
