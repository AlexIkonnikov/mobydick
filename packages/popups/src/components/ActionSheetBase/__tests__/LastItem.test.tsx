import {render} from '@testing-library/react-native';
import React from 'react';

import LastItem from '../LastItem';

describe('@npm/mobydick-popups/ActionSheetBase', () => {
  it('should renders correctly LastItem', () => {
    const {toJSON} = render(<LastItem />);

    expect(toJSON()).toMatchSnapshot();
  });
});
