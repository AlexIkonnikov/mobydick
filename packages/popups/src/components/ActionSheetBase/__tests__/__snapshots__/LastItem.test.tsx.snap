// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@npm/mobydick-popups/ActionSheetBase should renders correctly LastItem 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#EEEFF1",
      "borderBottomLeftRadius": 18,
      "borderBottomRightRadius": 18,
      "borderTopColor": "#D1D4DB",
      "borderTopWidth": 1,
      "justifyContent": "center",
      "marginBottom": 12,
      "marginHorizontal": 12,
      "minHeight": 75,
      "opacity": 1,
      "paddingHorizontal": 30,
    }
  }
>
  <Text
    allowFontScaling={true}
    style={
      Array [
        Object {
          "color": "#20242D",
          "fontFamily": "Inter-Regular",
          "fontSize": 27,
          "lineHeight": 33,
        },
        Object {
          "paddingVertical": 22.5,
        },
      ]
    }
  />
</View>
`;
